web/src/beta/components/SubTabButtonList/index.stories.tsx

```diff
@@ -0,0 +1,21 @@
+import { Meta, StoryObj } from "@storybook/react";
+
+import SubTabButtonList from ".";
+
+const meta: Meta<typeof SubTabButtonList> = {
+  component: SubTabButtonList,
+};
+
+export default meta;
+
+type Story = StoryObj<typeof SubTabButtonList>;
+
+export const Default: Story = {
+  args: {
+    items: [
+      { id: "1", name: "GIS", active: true },
+      { id: "2", name: "Story", active: true },
```

Comment:
Only one should be active at a time 🙏🏻

Reply:
If you doublecheck the useManageSwitchState it will never allow this to happen 👍🏻

Reply:
Fix completed.